src/command.c

```diff
@@ -90,7 +90,49 @@ gboolean command_search(Client *c, const Arg *arg)
 
 gboolean command_yank(Client *c, const Arg *arg, char buf)
 {
-    /* TODO no implemented yet */
+    /**
+     * This implementation is quite 'brute force', same as in vimb2
+     *  - both X clipboards are always set, PRIMARY and CLIPBOARD
+     *  - the X clipboards are always set, even though a vimb register was given
+     */
+
+    const char *uri = NULL;
+    char *yanked = NULL;
+
+    g_assert(c);
+    g_assert(arg);
+    g_assert(c->webview);
+    g_assert(arg->i == COMMAND_YANK_URI || arg->i == COMMAND_YANK_SELECTION);
+
+    if (arg->i == COMMAND_YANK_URI) {
+        if ((uri = webkit_web_view_get_uri(c->webview))) {
+            yanked = g_strdup(uri);
+        }
+    } else if (arg->i == COMMAND_YANK_SELECTION) {
+        // copy web view selection to clipboard
+        webkit_web_view_execute_editing_command(c->webview, WEBKIT_EDITING_COMMAND_COPY);
+        // read back copy from clipboard
+        yanked = gtk_clipboard_wait_for_text(gtk_clipboard_get(GDK_SELECTION_PRIMARY));
+    }
+
+    if(!yanked) {
+        return FALSE;
+    }
+
+    // store in vimb default register
```

Comment:
Thank you for providing patches for the webkit2 port of vimb. Can you please use the oldschool ANSI-C comments `/* */` also for single line comments.